src/app/workbench/WorkbenchGenericView.tsx

```diff
@@ -161,7 +162,16 @@ export function WorkbenchGenericView({
 
   return (
     <MosaicWindow<number> path={path} title={view.name} renderToolbar={() => header}>
-      <div id={view.id} className={`position-relative flex-column shadow bg-body workbenchView rounded flex-grow-1 ${mosaicDrag ? 'pe-none' : ''}`}>
+      <div
+        id={view.id}
+        className={`position-relative flex-column shadow bg-body workbenchView rounded flex-grow-1 ${mosaicDrag || ordino.isAnimating ? 'resizeCursor' : ''}`}
+      >
+        <AnimatingOverlay
+          color={ordino.colorMap[currentWorkbench.entityId]}
+          isAnimating={ordino.isAnimating || mosaicDrag}
+          iconName={viewPlugin?.desc?.icon ? viewPlugin?.desc?.icon : 'fas fa-bars'}
```

Comment:
Is the `'fas fa-bars` for the ranking view? Can we add this icon already when we register the view so that we can remove the if-condition here?

Reply:
the fa-bars was more of just a sensible default if there is no icon registered, which currently should not occur, but the icon variable in the config is optional so its definitely possible

Reply:
added a DEFAULT_ICON constant to make this more obvious